docs/source/tutorials/datasets.rst

```diff
@@ -0,0 +1,457 @@
+.. include:: ../substitutes.rst
+
+********
+Datasets
+********
+
+The |sdk| allows you to interact with the datasets you have added to |company|.
+A dataset has the attributes ``title``, ``description``, ``dataset_type``, and ``data_rows``
+
+.. code-block::
+
+    {
+        "title": "Example title",
+        "description": "Example description ..",
+        "dataset_type": "0",
+        "data_rows": [
+            {
+                "data_hash": "<data_hash>",
+                "data_title": "sample_data.mp4",
+                "data_type": "VIDEO"
+            }, {
+                "data_hash": "<data_hash>",
+                "data_title": "image-group-abcde",
+                "data_type": "IMG_GROUP"
+            },
+            ...
+        ]
+    }
+
+Note how the ``data_rows`` can both consist of videos and image groups.
+
+Below, you can find tutorials on how to interact with your datasets.
+
+
+Creating a dataset
+==================
+
+You can use the |sdk| to create a dataset.
+First, you need to create a :xref:`public-private_key_pair` for |company|.
```

Comment:
I don't think we need this line in the SDK docs as at this point we should assume users have installed and authenticated appropriately. Made sense in the web app docs, but not sure it does here.